modules/gitops-argo-cd-notification.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * argo-cd-custom-resource-properties.adoc
+
+:_content-type: PROCEDURE
+[id="argo-cd-notification_{context}"]
+= Enable notifications with Argo CD instance
+
+[role="_abstract"]
+Enable/disable the Argo CD notifications controller using a toggle within the Argo CD CR. By default, notifications are disabled. To enable notifications, set the `enabled` property to `true` in the `.yaml` file:
+
+[source,yaml]
+----
+  apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+spec:
+  notifications:
+    enabled: True
```

Comment:
Hi Debargo, it should be `enabled: true`

sample

```
``` yaml
apiVersion: argoproj.io/v1alpha1
kind: ArgoCD
metadata:
  name: example-argocd
spec:
  notifications:
    enabled: true
```
```